src/utils/_colors.scss

```diff
@@ -34,3 +38,15 @@
     @return mix(white, $color, nth($step-variations-positive, $step));
   }
 }
+
+// Dark Variations Function
+// Returns the darker variation of a color, as defined in color palette
+@function dark-variation($color) {
```

Comment:
These functions should be deprecated in favor of the `color-variations` one because it has specified all the positive and negative color variations